app/src/main/java/com/svetlanakuro/mdfirst/ui/AppThemeSaved.kt

```diff
@@ -0,0 +1,41 @@
+package com.svetlanakuro.mdfirst.ui
+
+import android.content.Context
+import androidx.appcompat.app.AppCompatActivity
+import com.svetlanakuro.mdfirst.R
+
+const val NAME_SHARED_PREFERENCE = "APP_THEME"
+const val appThemeKey = "APP_THEME_KEY"
+const val DefaultTheme = 0
+const val MoonTheme = 1
+const val SunTheme = 2
+
+class AppThemeSaved {
+
+    companion object {
+        fun savedThemeToStyleId(savedThemeNumber: Int): Int {
+            return when {
+                (savedThemeNumber == MoonTheme) -> R.style.Theme_MDfirst_Moon
+                (savedThemeNumber == SunTheme) -> R.style.Theme_MDfirst_Sun
+                else -> R.style.Theme_MDfirst
+            }
+        }
+
+        fun getSavedTheme(context: Context): Int {
+            val preferences = context.getSharedPreferences(
+                NAME_SHARED_PREFERENCE,
+                AppCompatActivity.MODE_PRIVATE
+            )
+            return preferences.getInt(appThemeKey, DefaultTheme)
+        }
+
+        fun setSavedTheme(context: Context, themeNumber: Int) {
+            val preferences =
+                context.getSharedPreferences(NAME_SHARED_PREFERENCE, Context.MODE_PRIVATE)
```

Comment:
лучше сделай перенос вот так

```suggestion
                val preferences =  context.getSharedPreferences(
                    NAME_SHARED_PREFERENCE, 
                    Context.MODE_PRIVATE,
                )
```